packages/rspack-test-tools/tests/Validation.test.js

```diff
@@ -8,7 +8,7 @@ describe("Validation", () => {
 				errors.push(...args);
 			};
 			try {
-				const rspack = require("@rspack/core");
```

Comment:
Do we keep a case to verify the default export?

Reply:
Yes, see https://github.com/web-infra-dev/rspack/blob/main/packages/rspack-cli/tests/api/type/js-api-cjs.test.ts#L1